src/test/java/com/snowflake/kafka/connector/SecurityTest.java

```diff
@@ -35,21 +35,6 @@ public void testRSAPasswordOutput() throws IOException,
     assert ! searchInLogFile(testPasswd);
   }
 
-  @Test
-  public void testRSPasswordOutput() throws IOException
-  {
-    String testPasswd = "TestPassword1234!";
-    String testUserName = "TestName";
-    Map<String, String> testConf = ConnectorConfigTest.getConfig();
-    testConf.put(SnowflakeSinkConnectorConfig.SCHEMA_REGISTRY_AUTH_CREDENTIALS_SOURCE, "USER_INFO");
-    testConf.put(SnowflakeSinkConnectorConfig.SCHEMA_REGISTRY_AUTH_USER_INFO,
-      testUserName + ":" + testPasswd);
-    Utils.validateConfig(testConf);
-    TestUtils.assertError(SnowflakeErrors.ERROR_0012,
-      () -> new SnowflakeAvroConverter().configure(testConf, false));
-    assert ! searchInLogFile(testPasswd);
-  }
-
```

Comment:
why remove?

Reply:
This test is not necessary, since everything related with schema registry is removed from connector. "value.converter.basic.auth.credentials.source", "value.converter.basic.auth.user.info" and many configurations defined [here](https://docs.confluent.io/current/schema-registry/connect.html) that starts with "value.converter" or "key.converter" will be directly sent to converters. Connectors don't need to handle those properties. 
Also since SnowflakeSinkConnectorConfig.SCHEMA_REGISTRY_AUTH_USER_INFO is removed, the connector will never have a chance to print the password out.